control/control.yaml

```diff
@@ -165,4 +165,16 @@ components:
           description: >-
             Protocol specific states
           type: string
-          enum: [start, stop]
\ No newline at end of file
+          enum: [start, stop]
+
+    Action.Request:
+      description: >-
+        Request to traffic generator for top level choice of protocol/traffic/action-group having the detail action to perform 
```

Comment:
rephrase the description to make this easier to understand.

Reply:
I have modified the description. But looks like this description block is not visible in ReDoc model.